### PR TITLE
added comments for code review

### PR DESCRIPTION
**Pull request by mscwilson**
code was clear and easy to follow. Found one possible edge case problem but I didn't get round to looking at the tests so I don't know.

### CODE REVIEW

File: lib/Van.rb [hunk 1]
```diff
@@ -8,7 +9,7 @@ def initialize
   end
 
   def deliver_bikes(garage)
-    @storage.length.times {garage.storage << @storage.pop}
+    @storage.length.times {garage.storage << @storage.pop} # is there a more efficient way of doing this? I rather like it but I feel like there should be a way to unload them all at once
```
Comment (RTurney) [inline lib/Van.rb:12]: use a push method for van.storage to push bikes to the garage in one go

File: lib/Bike.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 class Bike
     def working?
-      @broken ? (false):(true)
+      @broken ? (false):(true) # I see what you're going for here but it feels a bit complicated.
```
Comment (RTurney) [inline lib/Bike.rb:3]: try using <not@broken> instead for readability?

**Review by RTurney — COMMENTED** [review]
Changes for me to make myself.
